src/pages/react/training/index.js

```diff
@@ -20,22 +20,33 @@ import {
 import { Card } from 'src/components/elements'
 import { Breadcrumb } from 'src/components/navigation'
 import { REACT_BOOTCAMP, CONVINCE_THE_BOSS_PDF } from 'src/config/data'
+import { WHY_REACTJS_ACADEMY } from 'src/config/images.js'
+import { createSocialMetas } from 'src/components/utils'
+
+const metas = {
+  title: 'Learn React | React GraphQL Academy',
```

Comment:
@ric9176 isn't this "React Training | React GraphQL Academy". Row 14 in the spreadsheet